Marketingupdate

ladyleet opened the pull request.

Commit:
saving content for marketing update

Commit:
marketing copy update

taras commented on app/components/mentor-card.js:
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```
You don't need to create a js file if you're not making any changes to the component's JS. Pls, remove this file

ladyleet replied:
done

taras commented on app/components/nav-bar.js:
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```
Pls, remove this file

ladyleet replied:
done

taras commented on app/components/site-footer.js:
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```
Pls, remove this file

ladyleet replied:
done

taras commented:
Looks good, other than the few unnecessary files. Pls, remove those and I'll merge this

ladyleet committed:
Delete mentor-card.js

ladyleet committed:
Delete nav-bar.js

ladyleet committed:
Delete site-footer.js

ladyleet commented:
@taras r?

Commit:
about page and footer updates

ladyleet commented:
- updated the site footer with social links.
- updated the about page.
- installed coin clicks add-on for my own entertainment while updating site. will uninstall when it's ready to go live. :P

taras commented:
:+1:
